### DISCUSSION TITLE
What in the world is the version???

### DISCUSSION THREAD

**Discussion by Wjplatformer** [TomSchimansky/CustomTkinter#639] [Q&A]
So currently, textboxes, which was supposed to be a version 5.0.0 feature, had already been released! How about tabs? :( (also, my project is a search engine 😉). So Just wondering if it is a step by step feature, and not just one big feature release. But I contribute to the tabs feature, if nobody is working on it right now :pensive:. Because I'm really looking forward to this feature. Thanks!

**Comment by TomSchimansky** [comment]
You can download the develop branch and have a look at the tabview its already finished. But there is some work to do in the theme files before I can release version 5.0.0 and I'm very busy at the moment, but I will try to do it as fast as possible.
